### PR TITLE
Setup travis ci

### PR DESCRIPTION
**Pull request by michaelw85**
#24 Add configuration file. 

To get this working @anacierdem you can create an account and enable builds for this repository. travis-ci.org
If you want to show the build status in the readme please replace my account with yours.

image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 sudo: false
 
 os:
-  - osx
```
Comment (anacierdem) [inline .travis.yml]: Why did you remove osx?

Comment (michaelw85) [inline .travis.yml]: It was in the default, removed it because a build will take twice as long. I do not expect different behavior from node on osx/linux and since it will also not run in windows anyway I thought just linux would be enough. You can add it if you want to run on osx as well.
